utils/ort/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -28,13 +28,13 @@ import org.ossreviewtoolkit.model.yamlMapper
 
 class CopyrightStatementsProcessorTest : WordSpec({
     "process" should {
-        "return a result with items merged by owner and prefix, sorted by owner and year" {
-            val input = File("src/test/assets/copyright-statements.txt").readLines()
+        val statements = File("src/test/assets/copyright-statements.txt").readLines()
```

Comment:
I had moved the initialization of these variables out of the inner test for performance reasons: Due to the `invocations = 5` the files would otherwise be read on each iteration, which is unnecessary.